### ISSUE TITLE
Broken Link

### ISSUE THREAD

**Issue by yardsale8** [stevetjoa/stanford-mir#46]
The new website looks great.  Please note that the *NMF Audio Mosaicing* link is broken on the front page.

**Comment by stevetjoa** [comment]
Yes, sorry. I started the notebook and was busy making some demo improvements. I'll fix that.

**Comment by HuwCheston** [comment]
Cleaning up these old issues: since the front page has now been updated with `.ipynb` links, this has been fixed.
